src/TTextEdit.h

```diff
@@ -88,6 +88,7 @@ class TTextEdit : public QWidget
     int getRowCount();
 
     QColor mBgColor;
+    // position of cursor, in characters, across the entire buffer
```

Comment:
Is it? I though it was something about the currently *active* line in the buffer (i.e. an index into `(QStringList) TBuffer::lineBuffer` that returns the whole line as a `QString` and the index into the outer std::deque in `(std::deque<std::deque<TChar>>) TBuffer::buffer`...

Reply:
That's what it seems like to me